sdk/cognitivelanguage/azure-ai-language-conversations/tests/test_orchestration_direct.py

```diff
@@ -1,179 +0,0 @@
-# coding=utf-8
-# ------------------------------------
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-# ------------------------------------
-
-import pytest
-
-from azure.core.exceptions import HttpResponseError, ClientAuthenticationError
-from azure.core.credentials import AzureKeyCredential
-
-from testcase import (
-    ConversationTest,
-    GlobalConversationAccountPreparer
-)
-
-from azure.ai.language.conversations import ConversationAnalysisClient
-from azure.ai.language.conversations.models import (
-    AnalysisParameters,
-    AnalyzeConversationResult,
-    QuestionAnsweringParameters,
-    ConversationParameters,
-    OrchestratorPrediction,
-    QuestionAnsweringTargetIntentResult,
-    ConversationTargetIntentResult,
-    LUISTargetIntentResult
-)
-
-
-class OrchestrationAppDirectTests(ConversationTest):
-
-    @pytest.mark.skip(reason="internal server error!")
-    @GlobalConversationAccountPreparer()
-    def test_direct_kb_intent(self, conv_account, conv_key, orchestration_project):
-
-        # prepare data
-        query = "How do you make sushi rice?"
-        target_intent = "SushiMaking"
-        input = AnalysisParameters(
-            query=query,
-            direct_target=target_intent,
```

Comment:
I noticed that the models still include a `direct_target` option - do we still need a test/sample using it?

Reply:
will ping Farhad from service team regarding this also

Reply:
he explained how direct_target works
and i tried to update the samples but found a bug and reported it to the service team

Reply:
march preview apis are giving 500s
discussed this with people from service team yesterday Chong, Farhad, and Ashour
they asked me to skip it for this release

Reply:
Sounds good - thanks for investigating.